OTHER-DATABASES.md

```diff
@@ -0,0 +1,19 @@
+# Other databases
+
+Here is a list of some other Metamath databases that we know of
+but are not included in this repository:
+
+
+* @digama0 's [Dependent-Type Theory](https://github.com/digama0/dtt.mm/blob/master/dtt.mm) 
+* ~@sctfn 's older version of [New Foundations](https://github.com/sctfn/metamath-nf/blob/master/nf.mm) (this has since been moved into this repository)
+* @tirix 's [Q0 Logic](https://github.com/tirix/q0.mm)
+* @pbrosnan [Number Theory Game](https://github.com/pbrosnan/ntg) based on peano.mm, with additions.
```

Comment:
I know he was not a good James Bond, but doesn't he deserve an apostrophe nonetheless ?